src/main/java/banking/model/Account.java

```diff
@@ -0,0 +1,26 @@
+package banking.model;
+
+public class Account {
```

Comment:
Can you use Lombok for generating getters, constructors?

Reply:
fixed